packages/app-store-cli/src/core.ts

```diff
@@ -94,7 +94,7 @@ export const BaseAppFork = {
       slug: slug,
       type: `${slug}_${category}`,
       // TODO: Remove usage of imageSrc, it is being used in ConnectCalendars.tsx. After that delete imageSrc in all configs and from here
-      imageSrc: `icon.svg`,
+      // imageSrc: `icon.svg`,
```

Comment:
Why comment it our instead of removing it?

Reply:
To be just sure about it. I thought something might break seeing the above comment.

Reply:
I have removed the field in the latest commit.